lib/pacemaker/pcmk_injections.c

```diff
@@ -89,30 +90,46 @@ inject_transient_attr(pcmk__output_t *out, xmlNode *cib_node,
  *                             7 when interval_ms is 0, inject nothing)
  */
 void
-pcmk__inject_failcount(pcmk__output_t *out, xmlNode *cib_node,
+pcmk__inject_failcount(pcmk__output_t *out, cib_t *cib_conn, xmlNode *cib_node,
                        const char *resource, const char *task,
                        guint interval_ms, int rc)
 {
-    if (rc == 0) {
-        return;
+    char *name = NULL;
+    char *value = NULL;
 
-    } else if ((rc == 7) && (interval_ms == 0)) {
-        return;
+    int failcount = 0;
+    xmlNode *output = NULL;
 
-    } else {
-        char *name = NULL;
-        char *now = pcmk__ttoa(time(NULL));
+    CRM_CHECK(cib_node != NULL, return);
```

Comment:
all of the pointer arguments have to be non-NULL

Reply:
whoops